specification/liftrqumulo/resource-manager/Qumulo.Storage/preview/2022-10-12-preview/examples/FileSystems_CreateOrUpdate_MinimumSet_Gen.json

```diff
@@ -41,9 +41,7 @@
             "marketplaceSubscriptionStatus": "PendingFulfillmentStart"
           },
           "provisioningState": "Accepted",
-          "userDetails": {
-            "email": "aaaaaaaaaaaaaaaaaaaaaaa"
-          },
+          "userDetails": {},
           "delegatedSubnetId": "aaaaaaaaaa",
```

Comment:
We very much discourage the use of this kind of meaningless examples.

Reply:
@weidongxu-microsoft We have created a task internally to resolve this in future. Can PR be merged for now